beetmoverscript/src/beetmoverscript/constants.py

```diff
@@ -113,6 +113,14 @@
     "vpn": "pub/vpn/",
 }
 
+MAVEN_PRODUCT_TO_PATH = {
+    "appservices": "maven2/org/mozilla/appservices/",
+    "geckoview": "maven2/org/mozilla/geckoview",
```

Comment:
Nit: Missing trailing `/` which could hypothetically be harmful if one day we decided to publish a product maven library called something like `geckoview-foo` 

```suggestion
    "geckoview": "maven2/org/mozilla/geckoview/",
```

Reply:
Thanks for catching that; fixed.